### ISSUE TITLE
Events page scraping meetup events no longer works

### ISSUE THREAD

**Issue by neozenith** [newwwie/newwwie.com#50]
The event scraping API for meetup seems to have finally expired.  

I wonder if there is a way we could fix this? I think we need a Pro account.

**Comment by mtranter** [comment]
@neozenith I had a look at this last night.. The meetup.com GraphQL API can serve the data needed for this page, but there will be a CORS issue. I can't see a way around this without hosting a proxy service to make the call to the GraphQL API.

FWIW, an example of the GraphQL query is here

```
query getEventsBYGroup(urlname: String) {
  groupByUrlname(urlname: $urlname) {
    name
    urlname
    groupPhoto {
      id
      baseUrl
      preview
    }
    logo {
      id
      baseUrl
      preview
    }
    unifiedEvents {
      edges {
        node {
          title
          description
          dateTime
          eventUrl
          going
          maxTickets
          duration
          imageUrl
          venue {
            name
            lat
            lng
            address
            city
          }
          group {
            name
            urlname
          }
        }
      }
    }
  }
}
```

**Comment by neozenith** [comment]
I’d be happy to host the code. I’m just not a GraphQL dev. 

It could be good to have like a lambda scrape the necessary information on a nightly frequency and cache it in a public S3 bucket as JSON. 

Or would that still have the same CORS issue?

**Comment by neozenith** [comment]
This is now resolved via #51 and #53
